app/src/main/java/com/duckduckgo/app/pixels/remoteconfig/AndroidBrowserConfigFeature.kt

```diff
@@ -67,4 +68,14 @@ interface AndroidBrowserConfigFeature {
      */
     @Toggle.DefaultValue(false)
     fun optimizeTrackerEvaluationV2(): Toggle
+
+    /**
+     * This feature flag guards a fix for blob downloads
+     *
+     * @return always returns `true` for internal builds
+     * @return `true` when the remote feature is enabled.
+     */
+    @Toggle.DefaultValue(true)
+    @InternalAlwaysEnabled
```

Comment:
`@InternalAlwaysEnabled` do we want this? if we have to kill switch this, we want to kill switch it for internal testers too i think